lib/flight_scheduler/submission/array_task.rb

```diff
@@ -27,34 +27,42 @@
 
 module FlightScheduler::Submission
   class ArrayTask
+    attr_reader :allocation, :job, :task
+
```

Comment:
This is more a stylistic change then anything else. Generally I prefer to use `attr_reader` when defining static attributes within in a constructor. The rubcop style guide is suitable vague on this particular case, the closest I found is this:
https://github.com/rubocop-hq/ruby-style-guide#attr-family

As far as them being public, it doesn't really matter. For the most part I don't bother marking things as private unless I want to make a point.

The bigger problem here is `FlightScheduler::Submission::ArrayTask` violates the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter). The accessors are only required because `allocation` is a big complex object which requires unpacking. IMO its the wrong abstraction, however it is out of scope to deal with it now.

tl;dr They changed for stylistic reasons and I've made them private
https://github.com/openflighthpc/flight-scheduler-controller/pull/12/commits/6ceddb26647c57bf535c1e163668064f75dbb5c5